modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -266,8 +266,8 @@
 
 // Donation reward for Farsighted Nightlight
 /obj/item/clothing/mask/gas/nightlight
-	name = "FIR-36 Rebreather"
-	desc = "A close-fitting respirator designed by Forestfel Intersystem Industries and originally meant for Ixian Tajarans, the FIR-36 Rebreather is commonly used by Military and Civilian Personnel alike. It reeks of Militarism."
+	name = "Heavy Duty Rebreather"
+	desc = "An ultra heavy duty rebreather, often used by those who need them for medical reasons."
```

Comment:
you realize rebreathers are meant for exploration in hostile environments right? underwater, in space, in places with toxic air/gasses, they aren't used for medical stuff

Reply:
in a universe where some species can not breath our air, this might help them filter out whatever they need, or helps them get more air. Its a trope, look at things like most of the mass effect races or bain from batman.